### PR TITLE
Added my current OOTR discography

### PR DESCRIPTION
**Pull request by DetectiveSky**
If there are any too-spicy sequences let me know, I'll remove them.

### CODE REVIEW

**Comment by DetectiveSky** [comment]
I've put the timestamps in the commit message but now I realize i should put them here instead

LoD Sinking Old Sanctuary, 1:41, loop at 11s - Simon's Theme Fanfare is a fanfare - CotM Dance of Illusions, 0:48, loop from beginning - SotN Lost Painting, 1:42, loop from 0:15 - Dance of Death, 1:12, loop from 0:05 - Deja Vu, 0:57, loop from 4s - Going Alone, 1:04, loop from 0:09 - Zelda Ura Temple of Time, 0:54, loop from beginning

File: data/Music/Castlevania Series/Super Castlevania 4/DS612SimonsThemeFanfare.meta [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Super Castlevania 4 - Vital Soul Get
```
Comment (PurpleHato) [inline data/Music/Castlevania Series/Super Castlevania 4/DS612SimonsThemeFanfare.meta:1]: Super Castlevania IV - Vital Soul Get

File: data/Music/The Legend of Zelda Series/Ura Zelda/DS612TempleOfTimeUra.meta [hunk 2]
```diff
@@ -0,0 +1,4 @@
+The Legend of Zelda Ura - Temple of Time
```
Comment (PurpleHato) [inline data/Music/The Legend of Zelda Series/Ura Zelda/DS612TempleOfTimeUra.meta:1]: The Legend of Zelda: Ura - Temple of Time

**Review by PurpleHato — CHANGES_REQUESTED** [review]

**Comment by DetectiveSky** [comment]
Should be good to go now

**Comment by DetectiveSky** [comment]
Lost is 49s long and loops from the beginning

**Comment by DaruniasJoy** [comment]
Okay, one small Detail stops me from adding all this.

Could you do me the Favor and Rename your Files like this :

Dance Of Illusions by DS612.meta

and Fanfares like this : 
Fanfare - Simons Theme by DS612.meta

**Comment by DaruniasJoy** [comment]
access to the spreadsheet should be granted make sure to add the tracks pls :)

**Pull request merged**